src/app/view/ui/insertTextDialog.js

```diff
@@ -16,7 +16,8 @@ export default class InsertTextDialog {
         let ref = this;
         this.__acceptTextButton.onclick = function() {
             Registry.viewManager.activateTool("InsertTextTool");
-            Registry.text = document.getElementById("inserttext_textinput").value;
+            // Replace Registry.text
+            //Registry.text = document.getElementById("inserttext_textinput").value;
             ref.__dialog.close();
```

Comment:
This logic needs to move to the corresponding vue.js file

Reply:
I moved onclick and removed __acceptTextButton, but I cannot find anywhere where __acceptTextButton is referenced, so I believe it should still function.